tokens/models/tokens/base/tokens_base_erc20.sql

```diff
@@ -236,5 +236,6 @@ FROM (VALUES
     , (0x4ea71a20e655794051d1ee8b6e4a3269b13ccacc, 'stataBasUSDC', 6)  
     , (0xd3c66029f050b2c220d3b8566e50d0e8b3f83302, 'FLOW', 18)    
     , (0xafb89a09d82fbde58f18ac6437b3fc81724e4df6, 'DOG', 18)  
+    , (0x96419929d7949D6A801A6909c145C8EEf6A40431, 'SPEC', 18)  
```

Comment:
https://basescan.org/token/0x96419929d7949D6A801A6909c145C8EEf6A40431

Reply:
Thank you. Just to confirm, once this is merged and processed, DEX pools using this token will be correctly decoded (including data already in the past)?

Reply:
We may have its symbol correctly match, but not the `prices.usd`.

Reply:
Thanks for letting me know. We just need these 2 missing columns, will these work? 

<img width="1150" alt="image" src="https://github.com/duneanalytics/spellbook/assets/137088126/5ed7398e-732e-4220-a660-b53f9ab97bda">

Reply:
Hey folks, is there an update on this?

I have been trying to get the decoded data of this pool for a week :)